ConvertFuncs.ahk

```diff
@@ -323,30 +323,41 @@ Convert(ScriptString)
       ; lexikos says var=value should always be a string, even numbers
       ; https://autohotkey.com/boards/viewtopic.php?p=118181#p118181
       ;
-      else If RegExMatch(Line, "i)^([\s]*[a-z_][a-z_0-9]*[\s]*)=([^;]*)", &Equation)	; Thanks Lexikos
+;      else If RegExMatch(Line, "i)^([\s]*[a-z_][a-z_0-9]*[\s]*)=([^;]*)", &Equation)	; Thanks Lexikos
+      else If (RegExMatch(Line, "(?i)^(\h*[a-z_][a-z_0-9]*\h*)=([^;\v]*)", &Equation))
```

Comment:
Why `(?i)` instead of `i)`? Don't they just do the same thing?

Reply:
Yes... equivalent... I was testing in another app and just did not change it back to the AHK option. I rarely use \s since it can also match CRLF. But since this is separating the lines anyway, the whitespace edit really doesn't provide any difference. I had edited the function needle (to support nested parentheses), and got carried away by editing the others. lol. Feel free to change them back if you like.

Reply:
I do wish I knew about \h earlier, that's why I had `( |\t)`, to avoid matching newlines

Reply:
AFAIK, \h covers all horizontal whitespace, so I use it exclusively (where most people use \s). I also use \v when I only want to cover vertical whitespace. The only time I use \s is when I wish to cover both horizontal and vertical whitespace, so it can come in handy for things like multiline captures (which is on my list to tackle soon btw.).